code/datums/diseases/_MobProcs.dm

```diff
@@ -17,7 +17,7 @@
 	if(HasDisease(D))
 		return 0
 
-	if(count_by_type(viruses, /datum/disease/advance) >= 3)
```

Comment:
You need to include this in your changelog. If you're going to make engineered diseases no longer prevent other diseases, you might as well remove the entire `if` statement line here.

This effects gameplay, why _didn't_ you note this in the body of the PR?

Reply:
We'll that's just sneaky and frankly a bit off putting.